src/components/cards/comparison/ChangeItem.vue

```diff
@@ -1,28 +1,61 @@
 <!-- Visualizes a delta between two controls -->
 <template>
   <v-row>
-    <v-col class="pa-0" cols="1">
-      <v-card class="pa-2" outlined tile>
-        <slot name="name" />
-      </v-card>
-    </v-col>
-    <v-col class="pa-0" cols="5">
-      <v-card class="pa-2" outlined tile>
-        <slot name="old" />
-      </v-card>
-    </v-col>
-    <v-col class="pa-1" cols="1" align-self="center">
-      <v-icon> arrow_forward </v-icon>
+    <v-col cols="3" xs="3" sm="2" md="2" lg="1" xl="1" class="pa-0">
```

Comment:
If the breakpoint for `md` is the same as the one for `sm` then it can be omitted, same for `xl` since it is the same as lg. Also if `xs` value is the same as the number of cols then it can also be omitted (all the way up until a breakpoint that you actually specify a value for).